core/backend/src/IModelDb.ts

```diff
@@ -76,6 +76,25 @@ import { ElementLRUCache, InstanceKeyLRUCache } from "./internal/ElementLRUCache
 
 const loggerCategory: string = BackendLoggerCategory.IModelDb;
 
+/**
+ * Arguments for saving changes to the iModel.
+ * @public
```

Comment:
Should we be creating this new interface and the associated overloads public now rather than starting them off as alpha or beta so we have a chance to modify them?

Reply:
For that we need to add new method for `saveChanges()` that is marked as alpha. saveChanges() is public so any arg it get also need to be public.

Reply:
Is there no option to make one overload of the a different level than another?  Then make the combined one internal?